src/ontodia/diagram/paperArea.tsx

```diff
@@ -352,7 +352,7 @@ export class PaperArea extends React.Component<Props, State> {
                 left: this.area.scrollLeft,
                 top: this.area.scrollTop,
             };
-            this.setState(adjusted);
+            this.setState(adjusted, callback);
```

Comment:
Also don't forget to call `callback` in the else branch (when `samePaperProps` is true).